### PR TITLE
Included database connection instructions

### PR DESCRIPTION
**Pull request by pl77**
Added instructions for connecting to the included docker database, for those who are less familiar with Docker's internal container networking.

Apologies for adding another pull request so quickly.  It popped out at me when I was rereading the Docker instructions.

### CODE REVIEW

**Comment by Zuluru** [comment]
No problem. In looking it over again, I noticed you have instructions about how to keep the database and uploads from being lost. Is there a need to do something similar to preserve config changes, like `config/.env` or (if applicable) `config/app_local.php`?

**Comment by pl77** [comment]
Good point. I mirrored my entire volume into a separate "deployment" directory outside of the working path for just that purpose.  I'll quickly tweak the docker-compose file again to reflect that.  I just didn't want people setting up their github clone (i.e. working path/docker-compose) directory or a sub-directory under it as the volume, because it'd get overwritten whenever they did a git pull and/or cause operating issues for having the same file twice in the directory tree (such as spinning the container down and the regenerating it with both the fresh github code as well as the older code in the subdirectory). 

My explanation likely isn't making any sense, lol.  I'll just write up a suggestion for the readme.  Give me 10 minutes.

**Comment by pl77** [comment]
Okay should be better now.  Likely still way too wordy in the readme, so if you want to edit it down and make it more succinct, please, be my guest 👍

**Comment by Zuluru** [comment]
I have zero experience with Docker, so I'm trusting that what you're writing is accurate and useful. Maybe there should be a comment there that the Docker instructions are new, not thoroughly tested, unofficial, something along those lines.

**Comment by pl77** [comment]
Good idea.  I'll quickly tweak it.

**Comment by pl77** [comment]
okay, hopefully that keeps the main Zuluru app separate from you having to answer a bunch of docker implementation questions, lol.  Just ping me if someone is having issues or send them to my fork and I'll try to handle them.
